src/core/public/rendering/_base.scss

```diff
@@ -5,7 +5,6 @@
  */
 // SASSTODO: Naming here is too embedded and high up that changing them could cause major breaks
 #opensearch-dashboards-body {
-  overflow-x: hidden;
```

Comment:
Why is this change necessary?

Reply:
It's a known problem that sticky behavior doesn't work if we use `overflow: hidden` on the parent component. More information can be found at https://github.com/w3c/csswg-drafts/issues/865. I don't really know why this property was written, but it seems like it was just inherited from the old code on refactoring.

History of changes: 
1) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/b5767804c131dfecb6f588584ca6f6801b503beb#diff-b735d2f2b9d87d3791ea90563e79a98455f186249f531b068f629b46adb5b24cR9
2) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/1a99df72ba52581aef83d260d3ffc3238337b23e#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R10
3) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/e6ca5faa9c9d34c4a9528466428813f603b22d45#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R6
4) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/585a84cc9678bb14d1dccb66b40a06d57dfaaf77#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R5

Reply:
Makes sense, Im good with that